src/main/java/com/softserveinc/ita/pageobjects/CityModal.java

```diff
@@ -0,0 +1,19 @@
+package com.softserveinc.ita.pageobjects;
+
+import static com.codeborne.selenide.Selenide.$x;
+import static java.lang.String.format;
+
+public class CityModal extends MenuModal {
```

Comment:
this modal probably should not extend this menu modal, there is nothing to inherit here, maybe in future could be implemented abstract BaseModal, with some common features as close button or modal title